system/EntityCast/CastAsArray.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+
+namespace CodeIgniter\EntityCast;
+
+/**
+ * Class CastAsArray
+ *
+ * @package CodeIgniter\Entity\Cast
+ */
+
+class CastAsArray extends AbstractCast
+{
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function get($value, array $params = []) : array
```

Comment:
Do not add space before the colon.
```suggestion
	public static function get($value, array $params = []): array
```